doc/changes/latest.inc

```diff
@@ -212,6 +218,6 @@ API
 
 - :func:`mne.io.read_raw_brainvision` no longer raises an error when there are inconsistencies between ``info['chs']`` and ``montage`` but warns instead by `Joan Massich`_
 
-- Add ``update_ch_names`` parameter to :meth:`mne.io.Raw.set_montage` to allow updating the channel names based on the montage by `Joan Massich`_
+- Add ``update_ch_names`` parameter to :meth:`mne.io.eeglab.eeglab.RawEEGLAB.set_montage` to allow updating the channel names based on the montage by `Joan Massich`_
```

Comment:
The old ref was correct, the new one is probably broken (we do not document that class)

Reply:
I updated it 'cos I wanted to point only to the `RawEEGLAB` montage. It is not the same than the rest.

maybe I can just let it as it was.

Reply:
I would leave it. But if you want it to say RawEEGLAB use double backticks and don't try to link